gen/perunServicesInit.pm

```diff
@@ -50,7 +53,7 @@ my ($agent, $service, $facility, $servicesAgent, $directory, $tmp_directory, $tm
 sub init {
 
 	my ($facilityId, $facilityName, $local_data_file, $serviceName, $getDataType);
-	GetOptions ("facilityId|f=i" => \$facilityId, "facilityName|F=s" => \$facilityName, "data|d=s" => \$local_data_file, "serviceName|s=s" => \$serviceName, "getDataType|t=s" => \$getDataType) or die;
+	GetOptions ("consent-eval|c"=>\$CONSENT_EVAL,"facilityId|f=i" => \$facilityId, "facilityName|F=s" => \$facilityName, "data|d=s" => \$local_data_file, "serviceName|s=s" => \$serviceName, "getDataType|t=s" => \$getDataType) or die;
```

Comment:
```suggestion
	GetOptions ("consentEval|c=i"=>\$CONSENT_EVAL,"facilityId|f=i" => \$facilityId, "facilityName|F=s" => \$facilityName, "data|d=s" => \$local_data_file, "serviceName|s=s" => \$serviceName, "getDataType|t=s" => \$getDataType) or die;
```
we can specify the type here to integer and follow the camelcase convention

Reply:
Do we really want ‘=i’, since we only want consentEval to be either false (0) or true (1)? If I understand correctly, this would require us always to specify with ‘-c 1’ and allow other values than 0 and 1 as well.

Reply:
you're right, sorry, my bad. @mattjoke could you please remove it?

Reply:
On it